example/remote-data-blocks-example-code.php

```diff
@@ -35,7 +35,6 @@ function load_only_if_parent_plugin_is_active() {
 	}
 
 	require_once __DIR__ . '/airtable/elden-ring-map/register.php';
-	require_once __DIR__ . '/airtable/events/register.php';
```

Comment:
oh, yea, that'd do it. Was that intentional?

Reply:
Yes, intentional. Because in this PR:

> This API mirrors the "Event Planning" template from Airtable, but it is easily swappable with another API.